plugin-server/tests/shared/db.test.ts

```diff
@@ -303,4 +303,160 @@ describe('DB', () => {
             expect(publicJobs[jobName]).toEqual(jobPayload)
         })
     })
+
+    describe('person and group properties on events', () => {
```

Comment:
given these are db tests (not e2e) might make sense to mock Redis. You can see how I did it on #9550

Reply:
Wouldn't we want to have e2e tests too?